fip-6.md

```diff
@@ -0,0 +1,141 @@
+---
+fip: 6
+title: Provide ability to deactivate FIO Addresses and Domains
+status: Draft
+type: Functionality
+author: Casey Gardiner <casey@dapix.io>
+created: 2020-04-22
+updated: 
+---
+
+## Abstract
+This FIP implements the following:
+* Adds the ability for FIO Address owners the ability to deactivate stale or unused addresses.
+* Adds the ability for FIO Domain owners the ability to set their domain as expired.
+* Adds new API end points for FIO Domain deactivation and FIO Address burn. 
+* Adds new fees for FIO Domain deactivation and FIO Address burn.
+
+## Motivation
+Presently the FIO Blockchain only burns and removes old addresses and domains once they reach their expiration date. Added functionality to remove these addresses and domains at any given time gives the owners flexibility.
```

Comment:
Does this in any way impact the ability of domain owners to control the addresses registered on their domain? As in, can a domain owner burn an address on their domain owned by someone else?

Reply:
Domain owners should not have any control over the addresses registered on their domain. There is no parent/child relationship.